applications/mne_analyze/plugins/channelselection/channelselection.cpp

```diff
@@ -159,7 +159,9 @@ void ChannelSelection::handleEvent(QSharedPointer<Event> e)
 {
     switch (e->getType()) {
         case EVENT_TYPE::SELECTED_MODEL_CHANGED:
-            onModelChanged(e->getData().value<QSharedPointer<ANSHAREDLIB::AbstractModel>>());
+            if(e->getData().value<QSharedPointer<ANSHAREDLIB::AbstractModel> >()->getType() == ANSHAREDLIB_FIFFRAW_MODEL) {
```

Comment:
this will stop the channel selection from working with the average model/view. maybe use != ANSHAREDLIB_BEMDATA_MODEL instead.